nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/processor-details.jsp

```diff
@@ -103,6 +103,18 @@
                         <div class="clear"></div>
                     </div>
                     <div class="setting">
+                        <div id="details-execution-node-container" class="execution-node-setting">
```

Comment:
Awesome. Will do!

I will leave this PR open and I will issue another PR with this commit and any subsequent work. I'll close out both PRs once everything is merged in.

Thanks again!